content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -221,7 +221,7 @@ For a reference to old feature gates that are removed, please refer to
 | `WinDSR` | `false` | Alpha | 1.14 | |
 | `WinOverlay` | `false` | Alpha | 1.14 | 1.19 |
 | `WinOverlay` | `true` | Beta | 1.20 | |
-| `WindowsHostNetwork` | `false` | Alpha | 1.26| |
+| `WindowsHostNetwork` | `true` | Alpha | 1.26| |
```

Comment:
Can you point to why this detail is right? It's also OK to get a tech review from SIG Network.

Reply:
This is correct. The source code is here:

https://github.com/kubernetes/kubernetes/blob/release-1.27/pkg/features/kube_features.go#L1157